src/databricks/labs/ucx/config.py

```diff
@@ -124,6 +125,7 @@ def from_dict(cls, raw: dict) -> "MigrationConfig":
             warehouse_id=raw.get("warehouse_id", None),
             num_threads=raw.get("num_threads", 10),
             log_level=raw.get("log_level", "INFO"),
+            skip_tasks=raw.get("skip_tasks", []),
```

Comment:
I am not convinced we need it.

Reply:
I am. Most customers will start with Assessment only where the other tasks may come weeks and sometimes Months later.

Reply:
then we need to split the jobs and not skip tasks.

Reply:
I would like them to be able to start with a quick assessment not messing up with the other tasks. Then redeploy when they are ready, making sure they have the latest code.

Reply:
we can split the jobs with enough justification.

Reply:
Some customers may want to not deploy the group migration. Even look at it as a risk.
The assessment is a read only job and considered low risk.